core/camera/src/main/java/com/google/jetpackcamera/core/camera/ConcurrentCameraSession.kt

```diff
@@ -122,6 +124,32 @@ internal suspend fun runConcurrentCameraSession(
             }
         }
 
+        // update cameraState to mirror the current zoomState
+        launch {
+            primaryCamera.cameraInfo.zoomState.asFlow().filterNotNull().distinctUntilChanged()
+                .collectLatest { zoomState ->
+                    // TODO(): buggy race condition between our setZoomRatio call updating the ZoomState and camera startup
```

Comment:
Add a link to the android issue tracker with the bug for this condition inside the parenthesist of `TODO()`.